src/rulesets/Base/base.jl

```diff
@@ -26,6 +26,12 @@ function rrule(::typeof(real), z::Number)
     return (real(z), real_pullback)
 end
 
+# Conversions to Float
```

Comment:
These need tests 
Should be pretty simple
`test_scalar(f, 1.2)`
and maybe
`test_scalar(f, 1⊢1.7)`

for each of these